src/components/dashboard/dashboard.spec.ts

```diff
@@ -0,0 +1,277 @@
+                                        <h4 class="dashboard-run-info-subtitle">16:32 14min</h4>
+                                    </div>
+                                </div>
+                            </div>
+                        </div>
+                    </ux-dashboard-widget>
+                    <ux-dashboard-widget
+                        id="widget-C"
+                        name="Runs"
+                        [colSpan]="2">
+
+                        <div class="dashboard-widget-container">
+
+                            <h2 class="dashboard-widget-title">Runs</h2>
+
+                            <div class="dashboard-widget-content-list">
+
+                                <div class="dashboard-widget-content stacked">
+
+                                    <div class="dashboard-run-info">
+                                        <h3 class="dashboard-run-info-title">NV Test</h3>
+                                        <h4 class="dashboard-run-info-subtitle">ID Test Name</h4>
+                                    </div>
+
+                                    <div class="dashboard-run-info">
+                                        <h3 class="dashboard-run-info-title">PC_IRENA</h3>
+                                        <h4 class="dashboard-run-info-subtitle">Test Name</h4>
+                                    </div>
+
+                                    <div class="dashboard-run-info muted">
+                                        <h3 class="dashboard-run-info-title">16/01/2017</h3>
+                                        <h4 class="dashboard-run-info-subtitle">16:32 14min</h4>
+                                    </div>
+
+                                </div>
+
+                                <div class="dashboard-widget-content stacked">
+                                    <div class="dashboard-run-info">
+                                        <h3 class="dashboard-run-info-title">NV Test_22</h3>
+                                        <h4 class="dashboard-run-info-subtitle">ID Test Name</h4>
+                                    </div>
+
+                                    <div class="dashboard-run-info">
+                                        <h3 class="dashboard-run-info-title">PC_hiLA</h3>
+                                        <h4 class="dashboard-run-info-subtitle">Test Name</h4>
+                                    </div>
+
+                                    <div class="dashboard-run-info muted">
+                                        <h3 class="dashboard-run-info-title">16/01/2017</h3>
+                                        <h4 class="dashboard-run-info-subtitle">16:32 14min</h4>
+                                    </div>
+                                </div>
+                            </div>
+                        </div>
+                    </ux-dashboard-widget>
+
+                </ux-dashboard>
+    `
+})
+export class DashboardComponent {
+
+    options: DashboardOptions;
```

Comment:
Set `options = this.initialOptions`, to save you doing it in the test body and running change detection again.